package.json

```diff
@@ -279,29 +309,53 @@
 					"default": "",
 					"description": "Suffix for generics (can be left empty)"
 				},
-				"VhdlLinter.style.instantiationLabelPrefix": {
+				"VhdlLinter.style.outPrefix": {
 					"scope": "window",
 					"type": "string",
 					"default": "",
-					"description": "Prefix for instantiation labels (can be left empty)"
+					"description": "Prefix for outputs (can be left empty)."
 				},
-				"VhdlLinter.style.instantiationLabelSuffix": {
+				"VhdlLinter.style.outSuffix": {
 					"scope": "window",
 					"type": "string",
 					"default": "",
-					"description": "Suffix for instantiation labels (can be left empty)"
+					"description": "Suffix for outputs (can be left empty)."
+				},
+				"VhdlLinter.style.inPrefix": {
+					"scope": "window",
+					"type": "string",
+					"default": "",
+					"description": "Prefix for inputs (can be left empty)."
+				},
+				"VhdlLinter.style.inSuffix": {
+					"scope": "window",
+					"type": "string",
+					"default": "",
+					"description": "Suffix for inputs (can be left empty)."
+				},
+				"VhdlLinter.style.inoutPrefix": {
+					"scope": "window",
+					"type": "string",
+					"default": "",
+					"description": "A Prefix for inouts (can be left empty)."
 				},
-				"VhdlLinter.ports.outRegex": {
+				"VhdlLinter.style.inoutSuffix": {
 					"scope": "window",
 					"type": "string",
-					"default": "^o_",
-					"description": "A regular expression to match output ports."
+					"default": "",
+					"description": "Suffix for inouts (can be left empty)."
```

Comment:
maybe also keep the port defaults?

Reply:
thinking about it no default is probably good. Even though I dislike suddenly dropping default functionality.